lib/models/solid-host.js

```diff
@@ -77,11 +77,7 @@ class SolidHost {
     const serverHost = getHostName(this.serverUri)
     if (originHost === serverHost) return true
     if (originHost.endsWith('.' + serverHost)) return true
-    // Allow the user's own domain
-    const userHost = getHostName(userId)
-    if (originHost === userHost) return true
-    if (trustedOrigins.includes(origin)) return true
-    return false
+    return !!trustedOrigins.includes(origin)
```

Comment:
which by default included the user's own domain?

Reply:
Apparently not, given the test: is this intended?

Reply:
I tried to [reuse `ldp.getTrustedOrigins`](https://github.com/solid/node-solid-server/blob/fix/faulty-403/lib/ldp.js#L467), which includes the user's own domain if the server is a multiuser server.

I thought we could remove some redundant code, but maybe I should revert this change back, since others might not use `ldp.getTrustedOrigins`?